src/config/trongate.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+return [
+    /*
+    |--------------------------------------------------------------------------
+    | Countdown
+    |--------------------------------------------------------------------------
+    |
+    | Who doesn't love a good ol' countdown? Specify the time (in seconds)
+    | we should be counting down before redirection delay hits.
+    |
+    */
+    'countdown' => 5,
+
+    /*
+    |--------------------------------------------------------------------------
+    | Delay before redirection
+    |--------------------------------------------------------------------------
+    |
+    | Bask in the glory of Trongate for just a while longer before
```

Comment:
Please fix the comment styling, see: https://twitter.com/taylorotwell/status/1499392585894346754